dali/core/mm/mm_test_utils.h

```diff
@@ -163,6 +163,10 @@ class test_resource_wrapper<owning, security_check, memory_resource<kind, order>
       return this->upstream_->deallocate(p, b, a);
     }, ptr, bytes, alignment);
   }
+
+  virtual Context do_get_context() const noexcept {
```

Comment:
This cannot be marked `override` because the base class for `any_context` does not contain a virtual function of this name (for any context this function is never used, but it's here regardless to avoid excessive complexity).

Reply:
Every time I see a comment in review that justifies how the code is written I think it should probably be put into the code itself.

Reply:
Cannot agree more.

Reply:
Explain - yes.
Justifies - not really. That would be just noise _and_ it's likely to go stale should this function make its way back to the interface (which I'm arguing for).